Vagrantfile

```diff
@@ -50,6 +50,7 @@ Vagrant.configure("2") do |config|
     eval `opam config env`
     opam remote add xs-opam git://github.com/xapi-project/xs-opam
     opam install -y lwt_react depext camlp4
```

Comment:
Is this necessary?

Reply:
I can check if it builds without this line.

Reply:
It seems that the build worked without that line.